### PR TITLE
fix: ChatSearchBox component

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

**Review by alexis-kochetov — CHANGES_REQUESTED** [review]
Fix PRs should be branched from the latest release and target this release as well

File: src/dotnet/UI.Blazor.App/Components/ChatSearchBox/chat-search-box.css [hunk 1]
```diff
@@ -63,10 +68,11 @@
     @apply bg-transparent;
     @apply text-03;
 }
-.chat-search-box label {
-    @apply relative;
-    @apply flex-x;
-}
+/*.chat-search-box label {*/
```
Comment (frolyo) [inline src/dotnet/UI.Blazor.App/Components/ChatSearchBox/chat-search-box.css]: why did you comment it? Shouldn't it me removed?

**Review by frolyo — APPROVED** [review]

File: src/dotnet/UI.Blazor.App/Components/ChatSearchBox/chat-search-box.css (hunk 1 above)
Comment (iqmulator) [reply src/dotnet/UI.Blazor.App/Components/ChatSearchBox/chat-search-box.css]: oops, forgot to delete

Comment (crui3er) [inline src/dotnet/UI.Blazor.App/Components/ChatSearchBox/chat-search-box.css]: remove it

**Review by crui3er — APPROVED** [review]

**Pull request closed without merging**